themes/classic/_dev/webpack.config.js

```diff
@@ -22,70 +22,86 @@
  * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-var webpack = require('webpack');
-var path = require('path');
-var ExtractTextPlugin = require("extract-text-webpack-plugin");
+const webpack = require('webpack');
+const path = require('path');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
-var plugins = [];
-
-var production = true;
-
-if (production) {
-  plugins.push(
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        warnings: false
-      }
-    })
-  );
-}
-
-plugins.push(
-  new ExtractTextPlugin(
-    path.join(
-      '..', 'css', 'theme.css'
-    )
-  )
-);
-
-module.exports = {
-  entry: [
-    './js/theme.js'
-  ],
+let config = {
```

Comment:
The binary has been on my side. It would be great to add a note about this somewhere, because I was using the default binary provided by Ubuntu. :neutral_face: